src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -149,28 +149,26 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
     const children: SceneFlexItemLike[] = [];
 
     for (const option of options) {
-      if (option.value === ALL_VARIABLE_VALUE) {
+      const { value: optionValue } = option;
+      if (optionValue === ALL_VARIABLE_VALUE || !optionValue) {
         continue;
       }
 
-      const expr = getExpr(option.value!);
+      const query = buildLogVolumeQuery(getExpr(optionValue), {
+        legendFormat: `{{${optionValue}}}`,
+        refId: optionValue,
+      });
       const queryRunner = new SceneQueryRunner({
         maxDataPoints: 300,
         datasource: explorationDS,
-        queries: [
-          {
-            refId: option.value!,
-            expr,
-            legendFormat: `{{${option.label}}}`,
-          },
-        ],
+        queries: [query],
       });
-      let body = PanelBuilders.timeseries().setTitle(option.label!).setData(queryRunner);
+      let body = PanelBuilders.timeseries().setTitle(optionValue).setData(queryRunner);
```

Comment:
This is the same because `option.value` and `option.label`, right?

Reply:
Yeah, so the only place where label and value are not the same is with `All` option.
```
this.setState({
      fields: [
        { label: 'All', value: ALL_VARIABLE_VALUE },
        ...(logsScene.state.detectedFields?.map((f) => ({
          label: f,
          value: f,
        })) || []),
      ],
    });
```

 But in that case we are not building layout 
 `if (optionValue === ALL_VARIABLE_VALUE || !optionValue) {
        continue;
      }`